.travis.yml

```
@@ -46,5 +79,9 @@ jobs:
           branch: master
 stages:
   - build
+  - lint
+  - jest
   - test
-  - deploy
+  - danger
```

Could we move danger above test? Its another fail fast step.